test/manual/amp-next-page/1.0/amp-next-page.amp.html

```diff
@@ -285,8 +301,13 @@ <h2>Host page</h2>
           }
         ]
       </script>
-      <div amp-next-page-separator>
-        Read more
+      <div separator class="amp-next-page-default-separator">
```

Comment:
Add an example with a template that uses `script type="text/plain" template="amp-mustache"` as well.